lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,19 @@
+<%page expression_filter="h" args="notification_type, notification_icon, notification_message, hint_notification"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "partially-correct"
```

Comment:
Thank you-- this is much more aligned with the story! Can you modify the mock on the sandbox so that it shows how the Hint bar will render? I'd like to see what it looks like with the 2 buttons presented.